src/Microsoft.AspNet.WebHooks.Custom.SqlStorage/Microsoft.AspNet.WebHooks.Custom.SqlStorage.nuspec

```diff
@@ -6,9 +6,9 @@
     <title>Microsoft ASP.NET Custom WebHooks SQL Module</title>
     <authors>Microsoft</authors>
     <owners>Microsoft, aspnet</owners>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
-    <projectUrl>http://www.asp.net/</projectUrl>
-    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <licenseUrl>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <projectUrl>https://www.asp.net/</projectUrl>
```

Comment:
This is unique. I left it alone 'til we decide on my [question above](https://github.com/aspnet/WebHooks/pull/227/files#r162497646).

Reply:
I can't imagine any reason why this would need to be different. Let's make it match.